pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -179,6 +180,11 @@ func (gc *GarbageCollector) Sync(discoveryClient discovery.ServerResourcesInterf
 		// Get the current resource list from discovery.
 		newResources := GetDeletableResources(discoveryClient)
 
+		// Filter resources that have been uninstalled and reinstalled stopped since the last sync.
```

Comment:
reword this sentence, I'm having trouble parsing "uninstalled and reinstalled stopped"

Reply:
done